unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/PluginSystem/PluginSystemFactory.cs

```diff
@@ -20,7 +21,7 @@ public static PluginSystem Create()
             pluginSystem.Register(() => new SkyboxController());
             pluginSystem.Register(() => new GotoPanelPlugin());
             pluginSystem.Register(() => new ExperiencesViewerFeature());
-            pluginSystem.Register(() => new EmoteAnimationsPlugin(DataStore.i.emotes));
+            pluginSystem.Register(() => new EmoteAnimationsPlugin(DataStore.i.emotes, new EmoteAnimationLoaderFactory(), new WearableItemResolver()));
```

Comment:
This will have to be manually replicated in explorer-desktop right?

Reply:
Yes! how would you prefer to be notified about this kind of stuff?